errors/handler.go

```diff
@@ -0,0 +1,57 @@
+package errorhandler
+
+import (
+	errorhandler "github.com/ONSdigital/dp-import-reporter/handler"
```

Comment:
Feels weird to import a package under the same name as the current package... not sure the best approach, happy to have a chat about it